tei/manana-sera-otro-dia.xml

```diff
@@ -4864,13 +4855,14 @@
       </div>
       <div n="3" type="act">
         <div type="scene" resp="#rojas" n="1">
-          <head> JornadaIII </head>
-          <stage> Salen doña Beatriz, Juan y Inés con mantos. </stage>
-          <sp who="#juan">
-            <speaker> JUAN </speaker>
+          <head> Jornada III </head>
+          <stage> Salen doña Beatriz, Juana y Inés con mantos. </stage>
+          <sp who="#juana">
+            <speaker> JUANA </speaker>
             <l> ¿No me dirás qué es, señora </l>
             <l part="I"> tu pensamiento? </l>
           </sp>
+          <!--  -->
```

Comment:
This empty comment should probably removed.

Reply:
true; corrected